ulauncher/modes/extensions/extension_controller.py

```diff
@@ -233,6 +222,7 @@ async def update(self) -> bool:
                 raise
 
         await self.toggle_enabled(was_running)
+        lifecycle_events.emit("extensions:updated", self)
```

Comment:
This calls `extension_registry.updated()` which updates the registry to set this instance.

But if this instance was not already in the extension_registry, how did we get it?

Reply:
You're right, we don't need to listed for `updated` even in the ext. registry. I'll remove that